src/react-components/auth/SignInModal.js

```diff
@@ -2,18 +2,84 @@ import React, { useCallback, useState } from "react";
 import PropTypes from "prop-types";
 import { CloseButton } from "../input/CloseButton";
 import { Modal } from "../modal/Modal";
-import { FormattedMessage } from "react-intl";
-import { Button } from "../input/Button";
+import { FormattedMessage, useIntl, defineMessages } from "react-intl";
+import { CancelButton, NextButton, ContinueButton } from "../input/Button";
 import { TextInputField } from "../input/TextInputField";
 import { Column } from "../layout/Column";
+import { LegalMessage } from "./LegalMessage";
 
 export const SignInStep = {
   submit: "submit",
   waitForVerification: "waitForVerification",
   complete: "complete"
 };
 
-export function SubmitEmail({ onSubmitEmail, initialEmail, showPrivacy, privacyUrl, showTerms, termsUrl, message }) {
+export const SignInMessages = defineMessages({
+  pin: {
+    id: "sign-in.pin",
```

Comment:
Should these be `sign-in-modal` (and update the en.json, etc)? I believe this is your exact example in the documentation.

```suggestion
    id: "sign-in-modal.pin",
```

Reply:
Yup :sweat_smile: I meant to come back and fix this, thanks for catching it!